apps/src/constants.js

```diff
@@ -103,8 +103,13 @@ export const TestResults = {
   BETTER_THAN_IDEAL: 102,
 
   SUBMITTED_RESULT: 1000,
+  // LOCKED_RESULT and READONLY_SUBMISSION_RESULT are never stored on the backend.
+  // They are used to encode locked/readonly information into a TestResult. Eventually,
+  // we want to remove these and look at levelProgress.locked and levelProgress.readonly
```

Comment:
NIT: since it looks like you didn't update the name of `studentLevelProgressType` to `levelProgressType` in this PR, this comment would be slightly clearer if you say `studentLevelProgress.locked` / `studentLevelProgress.readonly`.